.code-review-bot/review_bot.py

```
@@ -1,3 +1,269 @@
+            return r.json()
+        except (requests.exceptions.ChunkedEncodingError, requests.exceptions.ProtocolError) as e:
+            last_exc = e
+            # exponential backoff
+            import time
+
+            time.sleep(min(2 ** attempt * 0.5, 5.0))
+            continue
+    # If we exhausted retries, raise the last exception or a generic HTTP error
+    if last_exc:
+        raise last_exc
+    raise RuntimeError("Unknown error posting to GitHub API")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)